cfgmgr/buffermgrdyn.cpp

```diff
@@ -887,7 +887,14 @@ void BufferMgrDynamic::refreshSharedHeadroomPool(bool enable_state_updated_by_ra
                           profile.speed.c_str(), profile.cable_length.c_str(), profile.port_mtu.c_str(), profile.gearbox_model.c_str());
             // recalculate the headroom size
             calculateHeadroomSize(profile);
-            updateBufferProfileToDb(name, profile);
+            if (task_process_status::task_success == doUpdateBufferProfileForSize(profile, false))
+            {
+                updateBufferProfileToDb(name, profile);
```

Comment:
updateBufferPoolToDb

Reply:
Actually, this is a redundant call. No need to update buffer pool either because it will be updated at the end of the function.